backend/src/pilstark/json_exporter/mod.rs

```diff
@@ -336,7 +335,9 @@ impl<'a, T: FieldElement> Exporter<'a, T> {
                             ..DEFAULT_EXPR
                         },
                     ),
-                    UnaryOperator::LogicalNot => panic!("Operator {op} not allowed here."),
+                    UnaryOperator::LogicalNot | UnaryOperator::Next => {
+                        panic!("Operator {op} not allowed here.")
+                    }
```

Comment:
Shouldn't `next` be supported here?

Reply:
No, the condenser must have combined it with the reference already.

We might even say that `next` is not supported on AlgebraicExpression and could add that to the type system (by using a different UnaryOperator).